src/NetworkViewUi.lua

```diff
@@ -163,45 +181,125 @@ function M.update_items(player_index)
   })
   item_flow.style.size = { width = M.WIDTH - 30, height = M.HEIGHT - 82 }
 
-  local rows = M.get_rows_of_items(net_view.view_type)
+  local h_stack_def = {
+    type = "flow",
+    direction = "horizontal",
+  }
+
+  -- pad the item row so there is a slot to dump items from the cursor
+  local did_pad = false
+  local function pad_item_row(item_h_stack, count)
+    if is_item then
+      for _ = 1, count do
+        item_h_stack.add({
+          type = "sprite-button",
+          sprite = "utility/slot_icon_resource_black",
+          tags = { event = UiConstants.NV_ITEM_SPRITE },
```

Comment:
Can this have a tooltip to explain what it is used for?

Reply:
Yes. What sort of wording?
Perhaps something like "click to crop the cursor inventory into the item network"?
Might be too wordy.

Reply:
I like the way that Factory Planner labels each key command on a separate line. I'd prefer to list every available key combination rather than try to keep the description short.

![image](https://github.com/year6b7a/item-network-factorio-mod/assets/135451123/69d4e6f0-6b1a-46e0-8429-a7817a2bc511)